src/datachain/lib/image.py

```diff
@@ -1,17 +1,44 @@
 from typing import Callable, Optional, Union
 
 import torch
-from PIL import Image
+from PIL import Image as PILImage
+
+from datachain.lib.file import File, FileError, Image, ImageFile
+
+
+def image_info(file: Union[File, ImageFile]) -> Image:
+    """
+    Returns image file information.
+
+    Args:
+        file (ImageFile): Image file object.
+
+    Returns:
+        Image: Image file information.
+    """
+    file = file.as_image_file()
+
+    try:
+        with file.open() as stream:
+            img = PILImage.open(stream)
+    except Exception as exc:
```

Comment:
Sounds like a broad catch. I am not sure about it, even though we reraise or wrap the exception. So, should be fine. Just thinking through.

Reply:
Good catch, let me think 👍

Reply:
Hm, I think it is OK here to catch general exception since we are going to show it to user.